src/models/FilterStakeholder.js

```diff
@@ -0,0 +1,148 @@
+// import { stringToSearchRegExp } from '../utilities';
+
+/*
+ * A simple model for grower filter
+ */
+
+export const ALL_ORGANIZATIONS = 'ALL_ORGANIZATIONS';
+export const ORGANIZATION_NOT_SET = 'ORGANIZATION_NOT_SET';
+
+export default class Filter {
+  constructor(options) {
+    Object.assign(this, options);
+  }
+
+  getWhereObj() {
```

Comment:
Are we using LoopBack query syntax with the new API?

Reply:
I wasn't sure if we should keep it uniform from the client, so I did, but we don't have to.  There is validation for what can be filtered on the API as well.